test/cpp/aot_inductor/CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ target_link_libraries(test_aot_inductor PRIVATE
   ${CMAKE_CURRENT_BINARY_DIR}/libaot_inductor_output.so
 )
 
+target_include_directories(test_aot_inductor PRIVATE
```

Comment:
With your change to setup.py, I suspect we don't need this now?

Reply:
We still need this. ```torch/_inductor``` is not in the header-file search paths.